getting-started/cloud-multi-node.md

```diff
@@ -0,0 +1,250 @@
+Cloud account with TimescaleDB 2.0 as a "do-it-yourself" (DIY) multi-node experience.
+
+## Overview of multi-node setup
+
+Multi-node clusters consist of at least two or more TimescaleDB instances 
+(called **Services** in Timescale Cloud). Each cluster has one access node (AN) 
+and one or more data nodes (DN). As outlined in our [architecture blog posts][distributed-architechture], 
+the access node is intended to be the only TimescaleDB instance that you or your 
+applications connect to once the cluster is set up. It becomes the "brains" and 
+traffic controller of all distributed hypertable activity. In contrast, data nodes 
+are not intended to be accessed directly once joined to a multi-node cluster.
+
+>:TIP:A proper TimescaleDB cluster should have at least two data nodes to begin 
+realizing the benefits of distributed hypertables. While it is technically possible
+to add just one data node to a cluster, this will perform worse than a
+single-node TimescaleDB instance and is not recommended. 
+
+### Step 1: Create Services for Access and Data node Services [](step1)
+
+First, you need to create new Services within your Cloud account. As mentioned 
+earlier, you should create _at least_ three Services to set up a multi-node cluster: 
+one access node and two data nodes. 
+
+There is currently no way to distinguish between the access node and data 
+nodes within the Timescale Cloud console, **so we strongly recommend that you include 
+“AN" and "DN" in the names of each service, respectively (eg. "an-mycluster", 
+"dn1-mycluster", "dn2-mycluster", etc.)**. Services can only assume one role in a 
+cluster (access or data node), and only one Service can act as the access node.
+
+For simplicity you can start with the same hardware configuration for all Services. 
+On Timescale Cloud, Service plans can be upgraded later to better tune access 
+and data node requirements.
+
+>:TIP:More advanced users might consider using larger disks on data nodes (this is 
+where the distributed hypertable data is stored) and more memory and CPU for the 
+access node.
+
+>:WARNING: To setup your first multi-node instance in Timescale Cloud, you will
+need to create new instances which will default to TimescaleDB 2.0, the version
+required to implement a multi-node cluster.
+
+### Step 2: Modify Access Node settings [](step2)
+
+The hard work of handling distributed queries in a multi-node cluster is handled
+by TimescaleDB for you. Some queries, however, will work better in a distributed
+environment when parameters allow TimescaleDB to more efficiently push down 
+some types of queries to the data nodes, or not have JIT interfere with a
+distributed planning process.
+
+To that end, we highly recommend the following settings be modified for the 
+**Access Node** only under the **Advanced Configuration** section.
+
+* `pg.jit` = off
+* `pg.max_prepared_transactions` > 0 (150 is a recommended starting value)
```

Comment:
I wonder if we should add here few other configuration options that can improve performances : eg using `rowbyrow` fetcher. One other is also enabling `enable_partitionwise_aggregate` so the work can be parallelized across data nodes. Not sure if I missed any. I guess you have a most recent list of tweaks you used during benchmarking?

Reply:
(seems I had a review in process - didn't realize - and lost some previous comments)

Cloud just doesn't support the other configuration options at this point. If/as they do, we'll update.